src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -159,14 +159,27 @@ private static object Locale_Parser(string name, PatternTable table)
                 }
```

Comment:
Maybe we make the method internal and then just add some tests. 
For example, we get the list of all cultures from the runtime and ensure all are parsed correctly?

That'd tell us if we ever need an update again. Depending on the runtime for a test probably a bit of anti-pattern, but we want to support all locale options that exist.